hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/Options.java

```diff
@@ -518,4 +518,28 @@ public String toString() {
     MD5MD5CRC,  // MD5 of block checksums, which are MD5 over chunk CRCs
     COMPOSITE_CRC  // Block/chunk-independent composite CRC
   }
+
+  /**
+   * Enum to support the varargs for swapBlockList() options.
+   */
+  public enum SwapBlockList {
+    NONE((byte) 0), // No options, swap the block list.
+    ONE_WAY_BLOCK_SWAP((byte) 1), // Skip replacing source with dst info.
+    EXCLUDE_BLOCK_LAYOUT_HEADER_SWAP((byte) 2);
```

Comment:
I think if we swap the block list we always want to swap the block layout header. Is there a situation where we would not want to do so?

Reply:
We don't have a use case currently. I will remove the Options altogether.